dictionary.json

```diff
@@ -2382,6 +2382,11 @@
       "description": "The data describing the DNS resource. The meaning of this data depends on the type and class of the resource record.",
       "type": "string_t"
     },
+    "record_id": {
+      "description": "Unique identifier for record or event",
+      "caption": "Record ID",
+      "type": "string_t"
+    },
```

Comment:
On second thought, maybe it won't cause merge conflicts. I think I'm experiencing more merge conflicts than normal because we're squash-merging, and I've created branches off of branches -- so things that should be automatic merges end up being marked as a conflict.

But naming conflicts could be an issue if OCSF adds an attribute with this name. I think you could call it `_record_id` to reduce the chances of that to almost zero. Plus it stands out as a bit unusual so we won't confuse it with something that's supposed to be in the upstream schema.

Reply:
Whoopsie! It should be in extensions's dictionary, should have caught it myself.